### ISSUE TITLE
installation says my OS X is too old but I have 10.4.11

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#6]
```
What steps will reproduce the problem?
1. download Soundflower-1.4.1.dmg and mount it.
2. open Soundflower.mpkg
3. within one second, the Installer says, "Operating System Too Old / Mac
OS 10.4 or higher is required to install Soundflower."

What is the expected output? What do you see instead?
I expected smooth installation.

What version of the product are you using? On what operating system?
Version 1.4.1 on OS X 10.4.11

Please provide any additional information below.
I am using ibook G4 1.33Ghz

```

Original issue reported on code.google.com by `yang...@gmail.com` on 17 Nov 2008 at 1:51

**Comment by GoogleCodeExporter** [comment]
```
I tried Soundflower 1.3.1 and that installed fine.
```

Original comment by `yang...@gmail.com` on 17 Nov 2008 at 2:08

**Comment by GoogleCodeExporter** [comment]
```
Same trouble here.  PowerBook G4 1.5GHz OSX 10.4.11.  I had SoundFlower 1.2.1 
previously installed and tried 
uninstalling before installing to 1.4.1.  Same error "Operating System Too 
Old..."
```

Original comment by `lewiskel...@gmail.com` on 17 Nov 2008 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
Same trouble here but with a MacBook Pro OSX 10.4.11: Same error "Operating 
System Too Old..."
```

Original comment by `f_ili...@bluewin.ch` on 20 Nov 2008 at 8:24

**Comment by GoogleCodeExporter** [comment]
```
This issue has been resolved in Soundflower 1.4.2, which is now available.
```

Original comment by `74obje...@gmail.com` on 24 Nov 2008 at 5:16
- Changed state: **Fixed**
